modules/nw-aws-nlb-existing-cluster.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-aws-network-load-balancer.adoc
+
+[id="nw-aws-nlb-existing-cluster_{context}"]
+= Configuring an Ingress Controller Network Load Balancer on an existing AWS cluster
+
+You can create an ingress controller backed by an AWS Network Load Balancer (NLB) on an existing cluster.
```

Comment:
I see Ingress Controller capitalized in some places and kept in lowercase in others. I would just cross-check for consistency,

Reply:
Oh no, what a silly mistake. Thank you!